plugins/pulp_rpm/plugins/importers/yum/listener.py

```diff
@@ -78,7 +79,11 @@ def download_succeeded(self, report):
         if isinstance(model, (models.RPM, models.SRPM)):
             self.metadata_files.add_repodata(model)
         # init unit, which is idempotent
-        unit = self.sync_conduit.init_unit(model.TYPE, model.unit_key, model.metadata, model.relative_path)
+        unit = self.sync_conduit.init_unit(model.TYPE, model.unit_key, model.metadata,
+                                           model.relative_path)
```

Comment:
Let's revert this change, and include it as a separate pull request to master since it it unrelated to the duplicate NEVRA.